src/azure-cli/azure/cli/command_modules/storage/tests/latest/test_storage_blob_live_scenarios.py

```diff
@@ -8,42 +8,50 @@
 from azure.cli.testsdk import (LiveScenarioTest, ResourceGroupPreparer, StorageAccountPreparer,
                                JMESPathCheck, JMESPathCheckExists, NoneCheck, api_version_constraint)
 from azure.cli.core.profiles import ResourceType
+from azure.cli.testsdk.decorators import serial_test
 
 
 @api_version_constraint(ResourceType.MGMT_STORAGE, min_api='2016-12-01')
 class StorageBlobUploadLiveTests(LiveScenarioTest):
+    @serial_test()
```

Comment:
Curious about it. Can we write something like this?
```
@StorageAccountPreparer()
@serial_test()
```

Reply:
`serial_test` is not like a decorator. Maybe I miss something 0.0

Reply:
It is a function, not a class. So I use lower case.

Reply:
> 
> 
> Curious about it. Can we write something like this?
> 
> ```
> @StorageAccountPreparer()
> @serial_test()
> ```

Good question. Let me try.